src/styles/ui-kit-buttons.module.scss

```diff
@@ -0,0 +1,71 @@
+@import '@styles/colors.scss';
+
+.container {
+    font-family: Wix Madefor Text;
+    width: 284px;
+    padding: 16px;
+}
+
+.uikit {
+    font-size: 10px;
+    font-weight: 700;
+    line-height: 12.6px;
+    letter-spacing: 0.1em;
+}
```

Comment:
Everything everywhere is in `px` so lets change the em to `1px`, it looks exactly the same